src/main/java/org/elasticsearch/action/admin/indices/segments/IndicesSegmentsRequest.java

```diff
@@ -37,7 +37,7 @@ public IndicesSegmentsRequest() {
 
     public IndicesSegmentsRequest(String... indices) {
         super(indices);
-        indicesOptions(IndicesOptions.fromOptions(false, false, true, false));
+        indicesOptions(IndicesOptions.fromOptions(false, true, true, false));
```

Comment:
Thanks a lot for checking! I did some digging too, to be honest I think this is a bug in `IndicesSegmentsRequest`. Shouldn't we just stick to the same default that comes from `BroadcastOperationRequest`? Would that differ from what you are doing? 

Also, maybe we could update the description of the PR to make it clearer to users that we are changing the default behaviour of indices segments api too.